fastjet/internal/delaunay/point.go

```diff
@@ -50,13 +51,21 @@ func (p *Point) inTriangle(t *Triangle) (inside, edge bool) {
 	barcen2 := ((t.C.Y-t.A.Y)*(p.X-t.C.X) + (t.A.X-t.C.X)*(p.Y-t.C.Y)) / det(t)
 	barcen3 := 1 - barcen1 - barcen2
 
-	// inside triangle
-	if barcen1 > 0 && barcen1 < 1 && barcen2 > 0 && barcen2 < 1 && barcen3 > 0 && barcen3 < 1 {
+	// use the math/big package to handle the geometric predicates
```

Comment:
implemented inTriangle using orientation. 

Where would I use OrientationKind in the code and did you make orientation and inCircumCircle public on purpose?